src/main/scala/io/iohk/ethereum/consensus/atomixraft/AtomixRaftConfig.scala

```diff
@@ -0,0 +1,71 @@
+package io.iohk.ethereum.consensus
+package atomixraft
```

Comment:
WDYT about moving this to `io.iohk.ethereum.utils`? I'm not saying it's best, but it would be consistent.

Also, the way we did `*Config` classes was just about holding the bare values: strings, numbers, durations. This holds object like `AtomixNode` or `File` and has logic to parse them.